core/ExceptionHandler.js

```diff
@@ -62,7 +68,19 @@ export class ExceptionHandler {
 
         this.logException(exception, options);
         if (options.showAlert) {
-            XH.appContainerModel.exceptionDialogModel.show(exception, options);
+            const alertType = options.alertType ?? ExceptionHandlerDefaults.ALERT_TYPE;
+            if (alertType === 'toast') {
+                XH.toast({
+                    message: exception.message,
```

Comment:
Think we do need a bit more structure, including the title "Exception" -- Raw Exception messages are just two varied and 

Can we have the toast show a styled component with a title of Exception, and then the message.  Think this might look a little more sane and controlled....